showcase/app/templates/components/form/index.hbs

```diff
@@ -0,0 +1,1078 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+{{page-title "Form Component"}}
+
+<Shw::Text::H1>Form / Layout components</Shw::Text::H1>
+
+<section data-test-percy>
+  <Shw::Text::H2>Containers</Shw::Text::H2>
+
+  <button type="button" class="shw-component-form-layout-button-highlight" {{on "click" this.toggleHighlight}}>
+    {{if this.showHighlight "Hide layout highlight" "Show layout highlight"}}
+  </button>
+
+  {{! Mix of shortcut names used for testing purposes }}
+  <div class="{{if this.showHighlight 'shw-component-form-layout-highlight-elements'}}">
+    <Shw::Grid @columns={{1}} @gap="2rem" as |SG|>
+      <SG.Item
+        @label="With Form, FormHeader, FormHeaderTitle, FormHeaderDescription, FormSection, SectionHeader, SectionHeaderTitle, FormSectionHeaderDescription, FormFields, FormSeparator, and ButtonSet"
+      >
+        <Hds::Form as |FORM|>
+          <FORM.Header>
+            <FORM.HeaderTitle>
+              <Shw::Placeholder @height="3.2em" @text="FormHeaderTitle" @background="#e5ffd2" />
+            </FORM.HeaderTitle>
+            <FORM.HeaderDescription>
+              <Shw::Placeholder @height="4.8em" @text="FormHeaderDescription" @background="#e5ffd2" />
+            </FORM.HeaderDescription>
+          </FORM.Header>
+
+          <FORM.Section as |FS|>
+            <FS.Header as |FSH|>
+              <FSH.Title>
+                <Shw::Placeholder @height="2.8em" @text="SectionHeaderTitle" @background="#add8e6" />
+              </FSH.Title>
+              <FORM.SectionHeaderDescription>
```

Comment:
```suggestion
                {{! Mix of shortcut names used for testing purposes }}
              <FORM.SectionHeaderDescription>
```

Reply:
I just thought it was maybe useful if someone came across the kind of weird mix of shortcut names I used here and wondered what was up with it...

Reply:
Removing locally